src/models/pieChart.js

```diff
@@ -208,8 +211,8 @@ nv.models.pieChart = function() {
         dispatch.tooltipShow(e);
     });
 
-    dispatch.on('tooltipShow', function(e) {
-        if (tooltips) showTooltip(e);
+    pie.dispatch.on('elementMouseout.tooltip', function(e) {
+        dispatch.tooltipHide(e);
     });
 
     dispatch.on('tooltipHide', function() {
```

Comment:
Should probably move this inside the block with the tooltipShow dispatch for consistency.